CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## 2.2.0-SNAPSHOT (YYYY-MM-DD)
 
+> [!WARNING]
+> This is the last version compatible with Sync feature.
```

Comment:
This is probably imprecise as I expect there may be more patch/minor versions we release in case we identify critical bugs that need to be addressed.